dev-libs/zziplib/Manifest

```diff
@@ -1,2 +1,2 @@
 DIST zziplib-0.13.71.tar.gz 1150654 BLAKE2B 00394de5da8b3c5b95ef0aea060e523354845261b846e3c876b13d2d05c6569496475a279d6b211c3215a6f17e6bfe53958baabcebb13cd72d30a9f0c40d9449 SHA512 e035d0ac26dca78335ae3defc652543ff7b353a1a95d76ed1beeb21a08e16f287a62d488f528cfbb77d5b558581b68d439aa0823577524e9aa61a3cf5f208cb5
-DIST zziplib-0.13.71_p20200419.tar.gz 1153032 BLAKE2B 34aa38fde43805636a83f34c064ab09880b7200c414d3666786b7d750a26d8f39dbb431dde7cb81e785cd20a0d444e2a7308b5c79eb71bf0e9f52a051fe01883 SHA512 7f94524ddd745d81656ea9b0314db791e636ed9a8305c4fde7f88572b0302bff4c7f8fa76ccf368b7b8b1c8062a4ad85dfab8f30cdf6ee37f835ff5758751222
+DIST zziplib-0.13.71_p20200419.tar.gz 1151538 BLAKE2B 1c0493284e092bd98af9fd9f3e470d019d5b15ccbd110b5fa9d1cfe2e40fff29d000fa364f4f9539397d2fa196287703795823d36273479bc5c7a34be0a3a52d SHA512 61b12fa8e316c43aaa2a2a328e8df7c757dc5eda01fe629af57341f51e7f063d8f7d0be3bdd4c6f1667acdccc9a110a822e7687de1c6a388493bce828729f8d3
```

Comment:
AFAIK the mirrors won't be updated unless you rename the tarball differently, just add something like `_upd.tar.gz` or so.

Reply:
replaced with p20201021